components/table/__tests__/Table.pagination.test.js

```diff
@@ -190,5 +190,7 @@ describe('Table.pagination', () => {
     expect(renderedNames(wrapper)[0]).toEqual('Jack');
     wrapper.find('.ant-pagination-item-2').simulate('click');
     expect(renderedNames(wrapper)).toEqual([]);
+    wrapper.setProps({ pagination: { current: 1, pageSize: 10, total: 100 } });
+    expect(renderedNames(wrapper)[0]).toEqual('Jack');
```

Comment:
Add another test case for this fix with different page.

Reply:
👌